tedana/workflows/tedana.py

```diff
@@ -102,10 +102,15 @@ def tedana(data, tes, mixm=None, ctab=None, manacc=None, strict=False,
 
     kdaw, rdaw = float(kdaw), float(rdaw)
 
+    try:
+        ref_label = os.path.basename(ref_img).split('.')[0]
+    except TypeError:
+        ref_label = os.path.basename(str(data)).split('.')[0]
```

Comment:
I believe `data` will always be a list, so should you be subsetting it before coercing to `str`?

Reply:
Testing locally, I'm only hitting the `TypeError` statement if we're provided with a non-list data type like `zcat` and the `ref_img` is nifti-like rather than a true string (as described in the `load_data` docstring). This is a good reminder to update that docstring !